packages/shared/src/dto/workflows/step-content-issue.enum.ts

```diff
@@ -2,7 +2,9 @@ export enum StepContentIssueEnum {
   MISSING_VARIABLE_IN_PAYLOAD = 'MISSING_VARIABLE_IN_PAYLOAD',
   ILLEGAL_VARIABLE_IN_CONTROL_VALUE = 'ILLEGAL_VARIABLE_IN_CONTROL_VALUE',
   MISSING_VALUE = 'MISSING_VALUE',
+  TIER_LIMIT_EXCEEDED = 'TIER_LIMIT_EXCEEDED',
```

Comment:
is this the correct word? will it be true to other things as well? or is it specific to the world of delay & digest

Reply:
i belive we could reuse it other aspects as well, 
```
          body: [
            {
              issueType: 'TIER_LIMIT_EXCEEDED',
              message:
                'The maximum body length allowed is 280 characters. Please contact our support team to discuss extending this limit for your use case.',
            },
          ]
```